extensions-core/datasketches/src/main/java/io/druid/query/aggregation/datasketches/quantiles/DoublesSketchDoubleAggregator.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.query.aggregation.datasketches.quantiles;
+
+import com.yahoo.sketches.quantiles.UpdateDoublesSketch;
+
+import io.druid.query.aggregation.Aggregator;
+import io.druid.segment.ColumnValueSelector;
+
+public class DoublesSketchDoubleAggregator implements Aggregator
```

Comment:
Consider renaming this class to `DoublesQuantileSketchAggregator`

Reply:
I am not sure about adding Quantiles into the class names. This is implied by the package name, and class names are long already. Regarding the second "Double", I believe the intention was to say that this aggregator works on double values as input (as opposed to sketches). I propose to rename it to BuildAggregator, and rename the other one to MergeAggregator (or leave Combining, but switch the words around for consistency)